README.md

```diff
@@ -346,6 +346,12 @@ The `usemin` task has 2 actions:
 * First it replaces all the blocks with a single "summary" line, pointing to a file creating by the transformation flow.
 * Then it looks for references to assets (i.e. images, scripts, ...), and tries to replace them with their revved version if it can find one on disk
 
+### Finding references
+
+`usemin` will look for different kinds of references in files of certain types. The type of the file should be indicated by using target name. That is to say, the target name in the `usemin` task should only be in `html`, `css`, `json` and `svg` so that `usemin` can find possible types of references in these files.
+
+For example, if the target name is `html`, `usemin` will find references of images, scripts, stylesheets, etc. However, `usemin` will only find references which are surrounded by `url(` and `)` when target name is `css`.
```

Comment:
Would it make more sense to use `url(...)` instead of `url(` and `)` as the placeholder?